tables/README.md

```diff
@@ -5,6 +5,10 @@ This module creates tables and supplementary tables for the manuscript
 Table 1: Molecular subtypes determined for this project
 - This table shows the type and number of different molecular subtypes profiled in the project.
 
+Table 2: Patients with hypermutant tumors
+- Listed are patients with at least one hypermutant or ultra-hypermutant tumor.
```

Comment:
Might want to indicate here that this includes all samples, not just primary  and including derived

Reply:
done